<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>800</width>
                <height>600</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MainWindow</string>
        </property>
        <widget class="QWidget" name="centralwidget">
            <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                    <widget class="QFrame" name="frame">
                        <property name="minimumSize">
                            <size>
                                <width>80</width>
                                <height>0</height>
                            </size>
                        </property>
                        <property name="maximumSize">
                            <size>
                                <width>80</width>
                                <height>16777215</height>
                            </size>
                        </property>
                        <property name="styleSheet">
                            <string notr="true">background-color: rgb(234, 234, 234);</string>
                        </property>
                        <property name="frameShape">
                            <enum>QFrame::StyledPanel</enum>
                        </property>
                        <property name="frameShadow">
                            <enum>QFrame::Raised</enum>
                        </property>
                        <layout class="QVBoxLayout" name="verticalLayout">
                            <item>
                                <widget class="QToolButton" name="gbl_btn_menu">
                                    <property name="minimumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>ArrowCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>目录</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="toolButtonStyle">
                                        <enum>Qt::ToolButtonTextUnderIcon</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="gbl_btn_my_analysis">
                                    <property name="minimumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>ArrowCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>我的分析</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="toolButtonStyle">
                                        <enum>Qt::ToolButtonTextUnderIcon</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="gbl_btn_public_data">
                                    <property name="minimumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>ArrowCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>公共数据</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="toolButtonStyle">
                                        <enum>Qt::ToolButtonTextUnderIcon</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="gbl_btn_mis">
                                    <property name="minimumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>ArrowCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>管理系统</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="toolButtonStyle">
                                        <enum>Qt::ToolButtonTextUnderIcon</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="gbl_btn_user_center">
                                    <property name="minimumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>ArrowCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>用户中心</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="toolButtonStyle">
                                        <enum>Qt::ToolButtonTextUnderIcon</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <widget class="QToolButton" name="gbl_btn_bi_tool">
                                    <property name="minimumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="maximumSize">
                                        <size>
                                            <width>60</width>
                                            <height>70</height>
                                        </size>
                                    </property>
                                    <property name="font">
                                        <font>
                                            <bold>true</bold>
                                        </font>
                                    </property>
                                    <property name="cursor">
                                        <cursorShape>ArrowCursor</cursorShape>
                                    </property>
                                    <property name="text">
                                        <string>BI工具</string>
                                    </property>
                                    <property name="icon">
                                        <iconset>
                                            <normaloff>icons/biaochaxun.png</normaloff>icons/biaochaxun.png
                                        </iconset>
                                    </property>
                                    <property name="iconSize">
                                        <size>
                                            <width>50</width>
                                            <height>50</height>
                                        </size>
                                    </property>
                                    <property name="toolButtonStyle">
                                        <enum>Qt::ToolButtonTextUnderIcon</enum>
                                    </property>
                                </widget>
                            </item>
                            <item>
                                <spacer name="verticalSpacer">
                                    <property name="orientation">
                                        <enum>Qt::Vertical</enum>
                                    </property>
                                    <property name="sizeHint" stdset="0">
                                        <size>
                                            <width>20</width>
                                            <height>61</height>
                                        </size>
                                    </property>
                                </spacer>
                            </item>
                        </layout>
                    </widget>
                </item>
                <item>
                    <widget class="QStackedWidget" name="mainContainer">
                        <property name="currentIndex">
                            <number>4</number>
                        </property>
                        <widget class="QWidget" name="page_menu"/>
                        <widget class="QWidget" name="page_my_analysis"/>
                        <widget class="QWidget" name="page_public_data"/>
                        <widget class="QWidget" name="page_mis"/>
                        <widget class="QWidget" name="page_user_center"/>
                        <widget class="QWidget" name="page_bi_tool"/>
                    </widget>
                </item>
            </layout>
        </widget>
        <widget class="QMenuBar" name="menubar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>800</width>
                    <height>22</height>
                </rect>
            </property>
        </widget>
        <widget class="QStatusBar" name="statusbar"/>
    </widget>
    <resources/>
    <connections/>
</ui>
